### PR TITLE
Feature/george/unit tests

### PR DESCRIPTION
**Pull request by zewolfe**

### CODE REVIEW

File: dhis2-android-app/src/test/java/DummyTestData.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
```
Comment (quesebifurcan) [inline dhis2-android-app/src/test/java/DummyTestData.java]: Would be nice if we can move all the test files into a test directory. That way, we can easily run all of them at once and the project structure will conform to (what I think is) Android standards.

(Move is as simple as `Refactor -> Move` if you're using Android Studio)

Comment (zewolfe) [inline dhis2-android-app/src/test/java/DummyTestData.java]: They are in a test directory. Switch from android view to project view in Android Studio. Hope that helps.

Comment (quesebifurcan) [inline dhis2-android-app/src/test/java/DummyTestData.java]: ah, got it. My mistake.

**Comment by quesebifurcan** [comment]
Looks good to me, +1

**Pull request merged**